src/Exception/RequestException.php

```diff
@@ -152,6 +158,36 @@ public static function getResponseBodySummary(ResponseInterface $response)
         return $summary;
     }
 
+    /**
+     * Define the amount of bytes to use when summarizing the response exception.
+     *
+     * @param int|null $size
+     *
+     * @return void
+     */
+    public static function setResponseSize($size)
+    {
+        if (is_null($size) || is_int($size)) {
+            self::$responseSize = $size;
+
+            return;
+        }
+
+        $message = '$size must be an integer with the amount of bytes before truncating or null to avoid truncation altogether';
+
+        throw new InvalidArgumentException($message);
+    }
+
+    /**
+     * Restore the amount of bytes to use when summarizing the response exception to it's default value.
+     *
+     * @return void
+     */
+    public static function restoreDefaultResponseSize()
```

Comment:
Is this needed?

Reply:
I think this could add a great DX, but I'm happy to take it out if you prefer.

Reply:
I dont see the scenario and it can easily be replaced with `$foo->setResponseSize(120)`.